src/index.ts

```diff
@@ -40,6 +40,7 @@ type Req = {
   method: string
   headers: Record<string, string>
   body: string
+  cache?: RequestCache
```

Comment:
This seems a bit silly, and not sure why it was done this way, but it seems `fetch()` itself takes a type `RequestInit` already which is a superset of this. With `fetch()` itself being defined as such: https://github.com/Microsoft/TypeScript/blob/v2.8.1/lib/lib.dom.d.ts#L15933

https://github.com/Microsoft/TypeScript/blob/v2.8.1/lib/lib.dom.d.ts#1403

Reply:
If I remember right, the types weren't available in Node at the time.